nose_unit.cfg

```diff
@@ -6,5 +6,5 @@ nologcapture=1
 verbosity=2
 where=yt
 with-timer=1
-ignore-files=(test_load_errors.py|test_load_sample.py|test_commons.py|test_ambiguous_fields.py|test_field_access_pytest.py|test_save.py|test_line_annotation_unit.py|test_eps_writer.py|test_registration.py|test_invalid_origin.py|test_outputs_pytest\.py|test_normal_plot_api\.py|test_load_archive\.py|test_stream_particles\.py|test_file_sanitizer\.py|test_version\.py|\test_on_demand_imports\.py|test_set_zlim\.py|test_add_field\.py|test_glue\.py|test_geometries\.py|test_firefly\.py|test_callable_grids\.py|test_external_frontends\.py|test_stream_stretched\.py|test_sanitize_center\.py|test_data_reload\.py)
+ignore-files=(test_load_errors.py|test_load_sample.py|test_commons.py|test_ambiguous_fields.py|test_field_access_pytest.py|test_save.py|test_line_annotation_unit.py|test_eps_writer.py|test_registration.py|test_invalid_origin.py|test_outputs_pytest\.py|test_normal_plot_api\.py|test_load_archive\.py|test_stream_particles\.py|test_file_sanitizer\.py|test_version\.py|\test_on_demand_imports\.py|test_set_zlim\.py|test_add_field\.py|test_glue\.py|test_geometries\.py|test_firefly\.py|test_callable_grids\.py|test_external_frontends\.py|test_stream_stretched\.py|test_sanitize_center\.py|test_data_reload\.py|test_image_comp_*\.py|test_line_plots\.py|test_mesh_slices\.py|test_raw_field_slices\.py|test_mesh_render\.py|test_vr_orientation\.py|test_profile_plots\.py|test_image_comp_geo\.py|test_image_comp_2D_plots\.py)
```

Comment:
Do you think we'll ever be able to break this into multiple lines? :)

Reply:
I wish. This one line generates so many conflicts that I have a nickname for it (I call it "the blockchain") ... I think I once tried splitting the argument and found that it wasn't supported. Supposedly it's possible to pass the equivalent command line argument more than once to combine regexes, but AFAIU this doesn't work with static configuration files. So if we replace this line by command line arguments, we may be able to escape conflict hell, but that'd be an entire mini project in itself.

Reply:
I'm addressing this in #4346